src/testdir/test_filetype.vim

```diff
@@ -422,6 +422,7 @@ let s:filename_checks = {
     \ 'robots': ['robots.txt'],
     \ 'rpcgen': ['file.x'],
     \ 'rpl': ['file.rpl'],
+    \ 'rsc': ['file.rsc'],
```

Comment:
@dkearns , it's already here, or am I missing something?

Reply:
Nope, it's there alright.  I assumed it would be in the second commit and didn't even check the first.

Sorry for wasting your time.